erts/emulator/sys/unix/sys_drivers.c

```diff
@@ -1046,6 +1046,16 @@ static void stop(ErlDrvData ev)
 	driver_select(prt, abs(dd->ofd->fd), ERL_DRV_USE, 0);  /* close(ofd); */
     }
 
+    if (dd->pid > 0) {
+        ErtsSysForkerProto *proto =
+            erts_alloc(ERTS_ALC_T_DRV_CTRL_DATA, sizeof(ErtsSysForkerProto));
+        memset(proto, 0, sizeof(ErtsSysForkerProto));
+        proto->action = ErtsSysForkerProtoAction_Stop;
+        proto->u.stop.os_pid = dd->pid;
+        erl_drv_port_control(forker_port, ERTS_FORKER_DRV_CONTROL_MAGIC_NUMBER,
```

Comment:
We don't need to be picky about whether this message is successfully delivered, I think the main cases not covered by the errno below have to do with the forker going down during VM halt and this patch specifically adds logic to the forker to kill processes and clean up even when Erlang cannot send the final messages.

It wouldn't be helpful to also crash Erlang faster in these cases, better to let the general flushing happen if it was requested.